src/Integrations/Greenhouse/Greenhouse.php

```diff
@@ -74,23 +84,35 @@ public function getForm(array $formAdditionalProps): string
 			return '';
 		}
 
+		// Return form to the frontend.
+		return $this->buildForm(
+			$this->getFormFields((string) $formId),
```

Comment:
You're casting this to string on line 82, so I don't think you need to do that here. Or just remove it on line 82, so this value will always be a string.

Come to think of it, remove it on line 82, as there you just casted the first expression of the ternary to string, then got something as a result that didn't have to be a string

https://3v4l.org/jC0XR